test/csuite/rwlock/main.c

```diff
@@ -50,8 +50,8 @@ void *thread_dump(void *);
 int
 main(int argc, char *argv[])
 {
-	TEST_OPTS *opts, _opts;
 	struct timespec te, ts;
+	TEST_OPTS *opts, _opts;
 	pthread_t dump_id, id[MAX_THREADS];
```

Comment:
my understanding is pthread_t declaration to be above struct timespec (alphabetical order).  is it not ?

Reply:
or is it because, it is not a struct ?

Reply:
> or is it because, it is not a struct ?

@bvpvamsikrishna, yes just because it's not a struct. Sorry, I wish there was a better explanation!